ArcGISRuntimeSDKQt_CppSamples/Layers/AddClusteringFeatureReductionToAPointFeatureLayer/AddClusteringFeatureReductionToAPointFeatureLayer.cpp

```diff
@@ -0,0 +1,244 @@
+{
+  // Add a class break for each intended value range and define a symbol to display for features in that range.
+  // In this case, the average building height ranges from 0 to 8 storeys.
+  // For each cluster of features with a given average building height, a symbol is defined with a specified color.
+  const QList<ClassBreak*> classBreaks =
+      {
+          new ClassBreak("0", "0", 0.0, 1.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(4, 251, 255), 8.0, this), this),
+          new ClassBreak("1", "1", 1.0, 2.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(44, 211, 255), 8.0, this), this),
+          new ClassBreak("2", "2", 2.0, 3.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(74, 181, 255), 8.0, this), this),
+          new ClassBreak("3", "3", 3.0, 4.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(120, 135, 255), 8.0, this), this),
+          new ClassBreak("4", "4", 4.0, 5.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(165, 90, 255), 8.0, this), this),
+          new ClassBreak("5", "5", 5.0, 6.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(194, 61, 255), 8.0, this), this),
+          new ClassBreak("6", "6", 6.0, 7.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(224, 31, 255), 8.0, this), this),
+          new ClassBreak("7", "7", 7.0, 8.0, new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColor(254, 1, 255), 8.0, this), this)
+      };
+
+  // Create a class breaks renderer to apply to the custom feature reduction.
+  // Define the field to use for the class breaks renderer.
+  // Note that this field name must match the name of an aggregate field contained in the clustering feature reduction's aggregate fields property.
+  ClassBreaksRenderer* classBreaksRenderer = new ClassBreaksRenderer("Average Building Height", classBreaks, this);
+
+  // Define a default symbol to use for features that do not fall within any of the ranges defined by the class breaks.
+  classBreaksRenderer->setDefaultSymbol(new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, QColorConstants::Svg::pink, 8, this));
+
+  // Create a new clustering feature reduction using the class breaks renderer.
+  m_clusteringFeatureReduction = new ClusteringFeatureReduction(classBreaksRenderer, this);
+
+  // Set the feature reduction's aggregate fields. Note that the field names must match the names of fields in the feature layer's dataset.
+  // The aggregate fields summarize values based on the defined aggregate statistic type.
+  m_clusteringFeatureReduction->aggregateFields()->append(
+      new AggregateField("Total Residential Buildings", "Residential_Buildings", AggregateStatisticType::Sum, this));
+
+  m_clusteringFeatureReduction->aggregateFields()->append(
+      new AggregateField("Average Building Height", "Most_common_number_of_storeys", AggregateStatisticType::Mode, this));
```

Comment:
stories?

Reply:
This is the name of the field in online data, it can't be changed here